src/tests/tests.py

```diff
@@ -0,0 +1,63 @@
+import os
+import sys
+from typing import Dict
+
+import pytest
+
+src_directory = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+
+sys.path.append(src_directory)
+
+from camera import Camera  # noqa: E402
+
+
+@pytest.fixture
+def sample_class():
+    return Camera()
+
+
+@pytest.mark.parametrize(
+    "details, expected_output",
+    [
+        (

```

Comment:
To make these bonkers strings more readable you could easily move them out into a test_constants.py file. Which will make your parametrization much simpler to read.